src/libs/OptionsListUtils.ts

```diff
@@ -884,10 +884,7 @@ function getReportOption(participant: Participant): OptionData {
         option.alternateText = translateLocal('workspace.common.workspace');
 
         if (report?.policyID) {
-            const policy = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${report.policyID}`];
-            const submitToAccountID = getSubmitToAccountID(policy, report);
-            const submitsToAccountDetails = allPersonalDetails?.[submitToAccountID];
-            const subtitle = submitsToAccountDetails?.displayName ?? submitsToAccountDetails?.login;
+            const subtitle = getChatRoomSubtitle(report, {isCreateExpenseFlow: true});
```

Comment:
I am just trying to understand if something can break with the case with`!subtitle || !config.isCreateExpenseFlow` ?

Reply:
You're right, I'm reverting that change